### DISCUSSION TITLE
New hardware: notepad Teclast F7

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#34] [Show and tell]
Not booting as legacy bios/CMS is no more available on recent computer even if this one is from 2018.
https://fr.pandacheck.com/blog/test-notebook-teclast-f7

I read that bootloader like https://wiki.osdev.org/Limine_Bare_Bones are compatible with x86...

**Comment by VendelinSlezak** [comment]
That's unfortunate. Problem is that BleskOS bootloader do not use BIOS only to load rest of code but also to get memory map, EDID and set graphic video mode, so for UEFI boot we need to rewrite whole BleskOS bootloader, and for now I have it as some point in future. Maybe next year I will do it so you then will be able to test BleskOS on this computer.
